test/Feature/InsertSpec.hs

```diff
@@ -411,15 +400,22 @@ spec actualPgVersion = do
       it "succeeds and returns usable location header" $ do
         let payload = [json| { "k":"圍棋", "extra":"￥" } |]
         p <- request methodPost "/simple_pk?select=extra,k"
-                     [("Prefer", "return=representation")]
-                     payload
+            [("Prefer", "tx=commit"), ("Prefer", "return=representation")]
```

Comment:
In this case, I thought it was important to still test the working location header. In other cases it's the same thing about the persistance of the changes made.

In those cases, I explicitely added a `Prefer tx=commit` **and** a cleanup request at the end (also with `tx=commit`).

Reply:
Agree, this one definitely needs persistence.